run_tests.sh

```diff
@@ -24,7 +24,7 @@ do
 
 	# build and run tests
 	if [ "$m" != '.' ]; then go build; fi
-	env GORACE="halt_on_error=1" go test -race -short ./...
+	env GORACE="halt_on_error=1" go test -race -short -count 1 ./...
```

Comment:
This was intended for local testing, to re-run all tests even if results were previously cached. I'm not sure if CI caches test results,  might it be necessary to ask CI to ignore cached results and do a more thorough testing? Will leave this here in the meantime to get opinions.

Reply:
I've never noticed a "(cached)", but this is fine I think.